File: labs/lab07/count_word.pl
#!/usr/bin/perl

$find = lc @ARGV[0];
$count = 0;

while ($line = <STDIN>) {
	chomp $line;
	$line = lc $line;
	@words = grep(/./, split(/[^a-zA-Z]/, $line));
	
	# loop through each line and check for the word
	foreach $i (@words) {
		if ($i =~ /^$find$/i) {
			$count++
		}
	}
}

print @ARGV[0], " occurred " , $count, " times\n";

File: labs/lab07/count_word.py
#!/usr/bin/python

import sys
import re

words = []

# loop through stdin
for lines in sys.stdin:
	words += re.split(r'[^a-zA-Z]', lines.lower());


words = filter(None, words)

count = 0;
for w in words:
	if w == sys.argv[1]:
		count += 1
		

# print output
print sys.argv[1] + " occurred " + str(count) + " times";
